File: codesignal/arcade/Intro/JS/adjacentElementsProduct.js
function adjacentElementsProduct(inputArray) {
    let max = inputArray[0] * inputArray[1];
    for(let i = 1; i < inputArray.length - 1; i++){
        if(max < inputArray[i] * inputArray[i+1]){
            max = inputArray[i] * inputArray[i+1]
        }
    }
    
    return max;
}

File: codesignal/arcade/Intro/Csharp/adjacentElementsProduct.cs
int adjacentElementsProduct(int[] inputArray) {
    int max = inputArray[0] * inputArray[1];
    
    for(int i = 2; i < inputArray.Length; i++){
        int num = inputArray[i] * inputArray[i-1];
        if(max < num) max = num;
    }
    return max;
}
